### PR TITLE
Best Practices 001 to 008

### PR DESCRIPTION
**Pull request by LukasBuecherl**
Added BP001 to BP008

### CODE REVIEW

File: SBOL/best-practices/BP002/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+> namespace: https://synbiohub.org/igem/2017_distribution  
+domain: https://synbiohub.org   
+root: igem/2017_distribution   
+collection: promoters/constitutive   
+displayId: BBa\_J23101
+
+SBOL-compliant URLs also facilitate auto-construction of child objects with unique *URLs*.
+Child objects of *TopLevel* objects with compliant *URLs* MUST conform to the following pattern:   
+\<parent_url\>/\<child_type\>\<child_type_counter\>" where the \<parent_url\> refers to the URL of the parent object, the \<child_type\> refers to the SBOL class of the child object, and \<child_type_counter\> is a unique index for the child object.
```
Comment (tcmitchell) [inline SBOL/best-practices/BP002/README.md:26]: The formatting gets wonky here. Can this be improved? Look at how this markdown renders on the branch.

File: SBOL/best-practices/BP006/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# BP 006 -- Recommended Ontologies for External Terms
+
+External ontologies and controlled vocabularies are an integral part of SBOL. SBOL uses *IRIs* (typically *URLs* to access existing biological information through these resources.
+New SBOL-specific terms are defined only when necessary.
+For example, *Component* *Component types*, such as DNA or protein, are described using Systems Biology Ontology (SBO) terms. Similarly, the *roles* of a DNA or RNA *Component* are described via Sequence Ontology (SO) terms. Although RECOMMENDED ontologies have been indicated in relevant sections where possible, other resources providing similar terms can also be used. A summary of these external sources can be found in \ref{tbl:preferred_external_resources}.
```
Comment (tcmitchell) [inline SBOL/best-practices/BP006/README.md:5]: The `\ref` at the end of this line looks like LaTeX. Can that be replaced by something else?

File: SBOL/best-practices/BP006/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+***
```
Comment (tcmitchell) [inline SBOL/best-practices/BP006/README.md:7]: This table does not render properly as a markdown table. Look at how this renders the branch. Can this be improved?

**Review by tcmitchell — CHANGES_REQUESTED** [review]
Some of the markdown formatting seems off. Take a look...

File: SBOL/best-practices/BP002/README.md (hunk 1 above)
Comment (LukasBuecherl) [reply SBOL/best-practices/BP002/README.md]: Updated as discussed to use forward quotes

File: SBOL/best-practices/BP006/README.md (hunk 2 above)
Comment (LukasBuecherl) [reply SBOL/best-practices/BP006/README.md]: Fixed that. Must have missed it the first time

File: SBOL/best-practices/BP006/README.md (hunk 3 above)
Comment (LukasBuecherl) [reply SBOL/best-practices/BP006/README.md:7]: Maybe. To be honest, I don't know how to render tables properly. Googling around I found external scripts to install but I wasn't sure how to do that

**Comment by cjmyers** [comment]
@jakebeal Can you have a look at BP 002?  In the PR to change from URI to IRI, you changed URI to URL in this best practice.  Should this have been changed to IRI?  If not, why?

File: SBOL/best-practices/BP006/README.md (hunk 3 above)
Comment (tcmitchell) [reply SBOL/best-practices/BP006/README.md:7]: See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#creating-a-table

**Review by tcmitchell — APPROVED** [review]

**Review by Gonza10V — APPROVED** [review]
Great work everyone

**Pull request merged**